templates/personal-settings.php

```diff
@@ -82,3 +83,13 @@
 		<?php endif; ?>
 	</div>
 </div>
+<div class="followupsection">
+	<h2><?php p($l->t('Server URL')); ?></h2>
```

Comment:
Should this be called "Server address" instead, which is also the phrase used in the clients?

Reply:
Agreed! :)